repo/installer.go

```diff
@@ -49,15 +65,25 @@ type Installer struct {
 	// of every file of every package, rather than only following imported
 	// packages.
 	ResolveAllFiles bool
-
-	// Updated tracks the packages that have been remotely fetched.
-	Updated *UpdateTracker
 }
 
-func NewInstaller() *Installer {
-	i := &Installer{}
-	i.Updated = NewUpdateTracker()
-	return i
+// NewInstaller returns a new Installer configured with the specified InstallerOptions.
+func NewInstaller(options InstallerOptions) *Installer {
+
+	return &Installer{
+		InstallerOptions: options,
+		Updated:          NewUpdateTracker(),
+		Vcs: NewVcs(VcsOptions{
+			Home:           options.Home,
+			UseCache:       options.UseCache,
+			UseCacheGopath: options.UseCacheGopath,
+			UseGopath:      options.UseGopath,
+			UpdateVendored: options.UpdateVendored,
+			Force:          options.Force,
+			Vendor:         options.Vendor,
+			Config:         options.Config,
+		}),
```

Comment:
Notice the `UpdateTracker` is attached to the `Installer` instead of the `Vcs`. In later calls it's looking for it as a property on `Vcs`. Can you make the tracker available in both places?